library/classes/rulesets/Amc/reports/AMC_314g_1_2_14_STG2/Numerator.php

```diff
@@ -39,5 +39,5 @@ public function test( AmcPatient $patient, $beginDate, $endDate )
 
         }
     }
-}
```

Comment:
This stuff is useless at this point anyway...but...I guess fixing it doesn't hurt!